resources/dicts/relationship_events/normal_interactions/admiration/decrease.json

```diff
@@ -1,152 +1,215 @@
+			"m_c scolds r_c for the way {PRONOUN/r_c/subject} acted during a meeting.",
+			"m_c and r_c argue over a piece of prey.",
+			"r_c panics over something that m_c finds trivial."
+		],
+		"reaction_random_cat": {
+			"admiration": "decrease"
+		}
+	},
+	{
+		"id": "admire_de_medium3",
+		"interactions": [
+			"m_c and r_c had to fix something around camp and argued about how to fix it. It ended up being fixed by some other cat in the end.",
+			"m_c notices that r_c has been avoiding {PRONOUN/r_c/poss} responsibilities lately.",
+			"m_c and r_c were supposed to be working together today, but they can never agree long enough to accomplish anything.",
+			"m_c is frustrated after finishing a task that r_c was supposed to do.",
+			"m_c pranked r_c during a patrol.",
+			"m_c is upset that r_c slept through patrol assignments.",
+			"m_c thinks that r_c acted disrespectfully during a patrol.",
+			"m_c noticed that r_c ignored an herb opportunity on a recent patrol.",
+			"After disagreeing on what to do during a recent patrol, m_c is frustrated with r_c."
+		],
+		"reaction_random_cat": {
+			"admiration": "decrease"
+		},
+		"main_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		]
+	},
+	{
+		"id": "admire_adult_de_med1",
+		"interactions": [
+			"m_c saw r_c miss an easy catch today.",
+			"m_c is disappointed in r_c's subpar work lately.",
+			"m_c is let down by r_c's sloppy performance during a sparring session.",
+			"r_c was supposed to be showing m_c a cool battle move but messed up so badly it made m_c cringe.",
+			"m_c noticed that r_c ignored an easy hunting opportunity on a recent patrol.",
+			"r_c froze up during a hunting patrol with m_c.",
+			"When m_c's border patrol encountered another clan, r_c acted like a fool.",
+			"m_c and r_c have very different approaches to hunting and argue about which is better."
+		],
+		"main_status_constraint": [
+			"apprentice",
+			"warrior",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior",
+			"deputy",
+			"leader"
+		]
+	},
+	{
+		"id": "admire_adult_de_med2",
+		"interactions": [
+			"m_c has noticed r_c isn't doing a lot of work lately.",
+			"m_c is disappointed in r_c's lack of initiative.",
+			"m_c notices that r_c seems to disappear whenever difficult tasks are being discussed.",
+			"r_c was meant to take a piece of prey to someone sick, but m_c saw {PRONOUN/r_c/object} eat it.",
+			"r_c didn't finish an important chore that m_c assigned {PRONOUN/m_c/object}.",
+			"The Clan notices that m_c and r_c won't talk to each other after coming back from a patrol."

```

Comment:
I think this should be moved to admire_adult_de_med1, as it's inappropriate for an m_c elder